api/client/src/main/java/org/projectnessie/client/auth/oauth2/AuthorizationCodeTokensRequest.java

```diff
@@ -39,10 +39,10 @@
 @Value.Immutable
 @JsonSerialize(as = ImmutableAuthorizationCodeTokensRequest.class)
 @JsonDeserialize(as = ImmutableAuthorizationCodeTokensRequest.class)
-interface AuthorizationCodeTokensRequest extends TokensRequestBase {
+interface AuthorizationCodeTokensRequest extends PublicTokensRequestBase {
 
   /** REQUIRED. Value MUST be set to "authorization_code". */
-  @Value.Default
+  @Value.Derived
```

Comment:
This change is because I realized that this property was available in builders, but it shouldn't because the value is fixed. AFAICT there is no annotation to say "exclude from builder, but keep serializing/deserializing it".